ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/TableExpireService.java

```diff
@@ -103,69 +101,94 @@ public void run() {
         if (!needClean) {
           return;
         }
-        long changeDataTTL = Long.parseLong(arcticTable.properties()
-            .getOrDefault(TableProperties.CHANGE_DATA_TTL,
-                TableProperties.CHANGE_DATA_TTL_DEFAULT)) * 60 * 1000;
-        long baseSnapshotsKeepTime = Long.parseLong(arcticTable.properties()
-            .getOrDefault(TableProperties.BASE_SNAPSHOT_KEEP_MINUTES,
-                TableProperties.BASE_SNAPSHOT_KEEP_MINUTES_DEFAULT)) * 60 * 1000;
-        long changeSnapshotsKeepTime = Long.parseLong(arcticTable.properties()
-            .getOrDefault(TableProperties.CHANGE_SNAPSHOT_KEEP_MINUTES,
-                TableProperties.CHANGE_SNAPSHOT_KEEP_MINUTES_DEFAULT)) * 60 * 1000;
-
-        Set<String> hiveLocation = new HashSet<>();
-        if (TableTypeUtil.isHive(arcticTable)) {
-          hiveLocation = HiveLocationUtils.getHiveLocation(arcticTable);
-        }
+        expireArcticTable(arcticTable);
+      } catch (Throwable t) {
+        LOG.error("unexpected expire error of table {} ", tableIdentifier, t);
+      }
+    }
+  }
 
-        if (arcticTable.isKeyedTable()) {
-          KeyedTable keyedArcticTable = arcticTable.asKeyedTable();
-          Set<String> finalHiveLocation = hiveLocation;
-          keyedArcticTable.io().doAs(() -> {
-            UnkeyedTable baseTable = keyedArcticTable.baseTable();
-            if (baseTable == null) {
-              LOG.warn("[{}] Base table is null: {} ", traceId, tableIdentifier);
-              return null;
-            }
-            UnkeyedTable changeTable = keyedArcticTable.changeTable();
-            if (changeTable == null) {
-              LOG.warn("[{}] Change table is null: {}", traceId, tableIdentifier);
-              return null;
-            }
+  public static void expireArcticTable(ArcticTable arcticTable) {
+    TableIdentifier tableIdentifier = arcticTable.id();
+    final String traceId = UUID.randomUUID().toString();
```

Comment:
What is the usage of this `traceId`?
It looks as if it is used to correlate related content when troubleshooting logs, and if so whether the table name would serve the same purpose.

Reply:
`traceId` is useless now, and should be removed.

It was used to trace a http request before, but now the orphan files clean can't be triggered from http request.